contrib/labstack/echo.v4/option.go

```diff
@@ -126,3 +128,14 @@ func WithHeaderTags(headers []string) Option {
 		cfg.headerTags = internal.NewLockMap(headerTagsMap)
 	}
 }
+
+// WithCustomTag will attach the value to the span tagged by the key. Standard
+// span tags cannot be replaced.
+func WithCustomTag(key string, value interface{}) Option {
+	return func(cfg *config) {
+		if cfg.tags == nil {
+			cfg.tags = make(map[string]interface{})
+		}
+		cfg.tags[key] = value
+	}
+}
```

Comment:
In v2 we are planning to implement a consistent pattern for `WithCustomTag` options, as described in #2203. In the meanwhile it's fine, but it may change.